app/views/licensees/index.html.erb

```diff
@@ -1,3 +1,32 @@
+
+<!-- <h3>Licensees</h3> -->
+
+<br>
+<%= link_to 'New Licensee', new_licensee_path, class: 'btn btn-primary' %>
+<br><br>
+
+<!-- <div class='card bg-light col-6 rounded mb-0'>
+  <div class='card-body'> -->
+
+
+    <%= link_to 'View All Licensees', licensees_path, class: 'btn btn-info mt-2' %>
+
+    <br>
+    <br>
+
+    <% @licensees.each do |l| %>
```

Comment:
For more clarity here, please use a variable name like `licensee` instead of a single letter variable name.